src/assets/bg_neon.svg

```diff
@@ -0,0 +1 @@
9,&quot;a&quot;:1.0},&quot;position&quot;:0.98000001907348633}],&quot;transform&quot;:{&quot;m00&quot;:555.06250,&quot;m01&quot;:-0.81292581558227539,&quot;m02&quot;:256.87518310546875,&quot;m10&quot;:0.99997699260711670,&quot;m11&quot;:252.24073791503906,&quot;m12&quot;:176.87965393066406},&quot;opacity&quot;:1.0,&quot;blendMode&quot;:&quot;NORMAL&quot;,&quot;visible&quot;:true}"/></g><g style="mix-blend-mode:color-dodge"><g clip-path="url(#d)" data-figma-skip-parse="true"><foreignObject width="4840.95" height="4840.95" x="-2420.47" y="-2420.47" transform="matrix(-.27753 .0005 .0004 .12612 1390 303.5)"><div xmlns="http://www.w3.org/1999/xhtml" style="background:conic-gradient(from 90deg,#f8f8f8 0deg,#000 0deg,#454545 299.423deg,#aaa 352.8deg,#f8f8f8 360deg);height:100%;width:100%;opacity:1"/></foreignObject></g><path d="M1818 0H962v607h856z" data-figma-gradient-fill="{&quot;type&quot;:&quot;GRADIENT_ANGULAR&quot;,&quot;stops&quot;:[{&quot;color&quot;:{&quot;r&quot;:0.97254902124404907,&quot;g&quot;:0.97254902124404907,&quot;b&quot;:0.97254902124404907,&quot;a&quot;:1.0},&quot;position&quot;:0.0},{&quot;color&quot;:{&quot;r&quot;:0.0,&quot;g&quot;:0.0,&quot;b&quot;:0.0,&quot;a&quot;:1.0},&quot;position&quot;:0.0},{&quot;color&quot;:{&quot;r&quot;:0.27145475149154663,&quot;g&quot;:0.27145475149154663,&quot;b&quot;:0.27145475149154663,&quot;a&quot;:1.0},&quot;position&quot;:0.83173078298568726},{&quot;color&quot;:{&quot;r&quot;:0.66666668653488159,&quot;g&quot;:0.66666668653488159,&quot;b&quot;:0.66666668653488159,&quot;a&quot;:1.0},&quot;position&quot;:0.98000001907348633}],&quot;stopsVar&quot;:[{&quot;color&quot;:{&quot;r&quot;:0.97254902124404907,&quot;g&quot;:0.97254902124404907,&quot;b&quot;:0.97254902124404907,&quot;a&quot;:1.0},&quot;position&quot;:0.0},{&quot;color&quot;:{&quot;r&quot;:0.0,&quot;g&quot;:0.0,&quot;b&quot;:0.0,&quot;a&quot;:1.0},&quot;position&quot;:0.0},{&quot;color&quot;:{&quot;r&quot;:0.27145475149154663,&quot;g&quot;:0.27145475149154663,&quot;b&quot;:0.27145475149154663,&quot;a&quot;:1.0},&quot;position&quot;:0.83173078298568726},{&quot;color&quot;:{&quot;r&quot;:0.66666668653488159,&quot;g&quot;:0.66666668653488159,&quot;b&quot;:0.66666668653488159,&quot;a&quot;:1.0},&quot;position&quot;:0.98000001907348633}],&quot;transform&quot;:{&quot;m00&quot;:-555.06250,&quot;m01&quot;:0.81292581558227539,&quot;m02&quot;:1667.1247558593750,&quot;m10&quot;:0.99997699260711670,&quot;m11&quot;:252.24073791503906,&quot;m12&quot;:176.87965393066406},&quot;opacity&quot;:1.0,&quot;blendMode&quot;:&quot;NORMAL&quot;,&quot;visible&quot;:true}"/></g></g><defs><clipPath id="c"><path d="M106 0h856v607H106z"/></clipPath><clipPath id="d"><path d="M1818 0H962v607h856z"/></clipPath><radialGradient id="a" cx="0" cy="0" r="1" gradientTransform="matrix(-670 0 0 -210.218 962 304)" gradientUnits="userSpaceOnUse"><stop offset=".154" stop-color="#fff" stop-opacity=".8"/><stop offset=".798" stop-color="#fff" stop-opacity="0"/></radialGradient></defs></svg>
```

Comment:
<img width="98" height="162" alt="Image" src="https://github.com/user-attachments/assets/b90be9ff-1d00-4637-89eb-91cbe15d8e6e" />

There is a weird line in the middle